refresh.template.py

```diff
@@ -241,6 +258,8 @@ def _get_files(compile_args: typing.List[str]):
                 # If we did ever go this route, you can join the output from aquery --output=text and --output=jsonproto by actionKey.
             # For more context on options and how this came to be, see https://github.com/hedronvision/bazel-compile-commands-extractor/pull/37
     compile_only_flag = '/c' if '/c' in compile_args else '-c' # For Windows/msvc support
+    if compile_only_flag not in compile_args:
+        raise ValueError(f"{compile_only_flag} not found in compile_args: {compile_args}")
     source_index = compile_args.index(compile_only_flag) + 1
```

Comment:
I think index will already throw a ValueError if not found!

Reply:
It just throws a KeyError, I think this exception will better let user know witch `compile_args` is missing.

Reply:
[At least on my system]
```Python
>>>"".index("foo")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: substring not found
```

But regardless, this is going to lead to a hard crash, right? Like this isn't something we expect to show the user.

Reply:
But w/o this two lines it still goes to a hard crash if both `/c` or `-c` not found...

Reply:
Totally agree!
It'll always say "-c not found...", though, bc of the previous line, though, right?
Not super opposed or anything, I'm not sure how much this adds.

Reply:
You're right! What about change typo to 
```python
if compile_only_flag not in compile_args:
  raise ValueError(f"/c or -c (required for parsing sources) is not found in compile_args: {compile_args}")
```

Reply:
Sure, but I really think this is okay without! They should always be present from Bazel, and we'd get a crash that user will report

Reply:
OK, I will just delete them. Maybe I only encounter during my development.

Reply:
Oh! I'm silly. You're adding it so that the contents of the array itself will be in the error message for easy debugging, right? That's awesome, because then people will actually report the compile command as part of the error.

(Sorry I was slow to understand.)

Reply:
^ To be clear, if easy, could you re-add your code from https://github.com/hedronvision/bazel-compile-commands-extractor/pull/49#discussion_r869977295. Again, my apologies for being slow to understand what you were up to.